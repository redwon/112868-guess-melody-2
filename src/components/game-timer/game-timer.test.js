import React from 'react';
import renderer from 'react-test-renderer';

import GameTimer from './game-timer';

it(`renders correctly`, () => {
  const tree = renderer
    .create(<GameTimer
      status={true}
      time={300000}
      onTick={jest.fn()}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
