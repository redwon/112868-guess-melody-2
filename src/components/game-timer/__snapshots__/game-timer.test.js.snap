// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="timer__value"
>
  <span
    className="timer__mins"
  >
    05
  </span>
  <span
    className="timer__dots"
  >
    :
  </span>
  <span
    className="timer__secs"
  >
    00
  </span>
</div>
`;
